Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityDeviceManager.cs

```diff
@@ -607,7 +625,8 @@ private void GestureRecognizer_HoldCompleted(HoldCompletedEventArgs args)
             var controller = GetController(args.source, false);
             if (controller != null)
             {
-                MixedRealityToolkit.InputSystem.RaiseGestureCompleted(controller, holdAction);
+                IMixedRealityInputSystem inputSystem = Service as IMixedRealityInputSystem;
+                inputSystem.RaiseGestureCompleted(controller, holdAction);
```

Comment:
>inputSystem [](start = 16, length = 11)

Unrelated to this change, but do you know why there's only a ? check on the first function (i.e. GestureRecognizer_HoldStarted but none here or below - i.e. I would assume that GestureRecognizer_ManipulationStarted would need something similar if the issue is only on starts)

Reply:
good question. i'll be sure to do a thorough scrub in the near future to ensure we are doing the right thing